src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -509,6 +509,21 @@ public function iShouldHaveEmptyCart(string $tokenValue): void
         Assert::same(count($items), 0, 'There should be an empty cart');
     }
 
+    /**
+     * @Then I should be unable to add it to the cart
+     */
+    public function iShouldBeUnableToAddItToTheCart(): void
+    {
+        /** @var ProductVariantInterface $productVariant */
+        $productVariant = $this->sharedStorage->get('productVariant');
+
+        $tokenValue = $this->pickupCart();
+        $this->putProductVariantToCart($productVariant, $tokenValue);
+
+        $response = $this->cartsClient->getLastResponse();
+        Assert::same($response->getStatusCode(), 422);
```

Comment:
can you check here validation message instead status code?

Reply:
I'd like to explain myself why I check status Code; the step is about 'unable to add it to the cart' and I dont think that this step cares why it happens (what validation message). This I think should be checked somewhere else, in add to cart validation behats 😃